website/client/src/pages/settings/components/yourBalance.vue

```diff
@@ -11,6 +11,7 @@
       class="balance-info"
       :currency-needed="currencyNeeded"
       :amount-needed="amountNeeded"
+      neededCurrencyOnly
```

Comment:
huh, does this work as a destructured syntax? Do we set up props like this anywhere else? I'd lean toward keeping the style consistent for readability, tbh, given you've used our more common format even within this PR.

Reply:
It does, which is weird. I have, however, rewritten it so it matches our more common format.